source/guide_guillotine.rst

```diff
@@ -0,0 +1,279 @@
+  Successfully installed guillotine-1.4.2
+  Fetching sequel-5.22.0.gem
+  Successfully installed sequel-5.22.0
+  Fetching mysql2-0.5.2.gem
+  Building native extensions. This could take a while...
+  Successfully installed mysql2-0.5.2
+  8 gems installed
+
+  [isabell@stardust ~]$
+
+Create App File
+---------------
+
+Create a directory, and inside the directory a file ``app.rb`` for the application:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mkdir ~/guillotine
+  [isabell@stardust ~]$ touch ~/guillotine/app.rb
+  [isabell@stardust ~]$
+
+Add the following content to ``~/guillotine/app.rb`` to set up the connection to the MySQL database using the Sequel adapter. Make sure to adapt your MySQL credentials.
+
+.. code-block:: ruby
+ :emphasize-lines: 6
+
+  require 'guillotine'
+  require 'sequel'
+
+  module MyApp
+    class App < Guillotine::App
+      db = Sequel.connect("mysql2://isabell:MySuperSecretPassword@127.0.0.1:3306/isabell")
+      adapter = Guillotine::Adapters::SequelAdapter.new(db)
+      set :service => Guillotine::Service.new(adapter)
+
+      post "/" do
+        status, head, body = settings.service.create(params[:url], params[:code])
+
+        if loc = head['Location']
+          body = head['Location'] = File.join(request.base_url, loc)
+        end
+
+        # Show shortened URL in Body
+        [status, head, simple_escape(body)]
+      end
+    end
+  end
+
+.. note:: This file can be used as a configuration point.
+  The Katana_  project provides an example for possible extensions.
+
+Create Rackup config file
+-------------------------
+
+Add the following content to ``~/guillotine/config.ru``:
+
+.. code-block:: ruby
+
+  require "rubygems"
+  require File.expand_path("../app.rb", __FILE__)
+
+  run MyApp::App
+
+
+Set up Database
+---------------
+
+Initialize the database for Guillotine:
+
+.. code-block:: console
+ :emphasize-lines: 1-9
+
+  [isabell@stardust ~]$ mysql <<__SQL__
+  USE $USER;
+  CREATE TABLE IF NOT EXISTS \`urls\` (
+    \`url\` varchar(255) DEFAULT NULL,
+    \`code\` varchar(255) DEFAULT NULL,
+    UNIQUE KEY \`url\` (\`url\`),
+    UNIQUE KEY \`code\` (\`code\`)
+  ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+  __SQL__
+  [isabell@stardust ~]$
+
+Configure web server
+--------------------
+
+.. note::
+
+    Guillotine is running on port 9292.
+
+.. include:: includes/web-backend.rst
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/guillotine.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:guillotine]
+  environment=APP_ENV=production
+  directory=%(ENV_HOME)s/guillotine
+  command=/opt/uberspace/etc/%(ENV_USER)s/binpaths/ruby/rackup config.ru --host 0.0.0.0 --port 9292
+
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+Create short URL
+================
+
+Use your HTTP tool of choice (e.g. ``curl``) to make a POST request to create a short URL.
+
+.. code-block:: console
+ :emphasize-lines: 1-2
```

Comment:
This breaks the highlighting of `[isabell@stardust ~]$`, so I am thinking to maybe remove the `emphasize-lines`.
Here is a screenshot of both versions side-by-side. What do you think?

<img width="1122" alt="image" src="https://user-images.githubusercontent.com/6382716/61996120-0f99e080-b091-11e9-9f00-11daab0c3f50.png">

Reply:
`emphasize-lines` should work just fine with highlighting, like in [the pgsql guide](https://lab.uberspace.de/guide_postgresql.html). Try specifying `1,2` instead of `1-2`.

Reply:
Same result for me, sorry.

Reply:
I'd go with removing `:emphasize-lines:` in all instances but `~/guillotine/app.rb`.